### PR TITLE
Feature/177 disable items in dropdownfield

### PR DESCRIPTION
**Pull request by lestard**
### Prerequisites

Please make sure you can check the following two boxes:

- [x] I have read the **CONTRIBUTING** document
- [x] My code follows the code style of this project

### Contribution Type

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [ ] Bug fix (non-breaking change which fixes an issue, please reference the issue id)
- [x] New feature (non-breaking change which adds functionality, make sure to open an associated issue first)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] My change requires a change to the documentation
- [x] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed

### Description

Add prop `disabledItems` to `DropdownField` and `InteractiveList` which takes a list of key that should be disabled so that users can't select them.
See #177

### CODE REVIEW

File: src/components/InteractiveList/InteractiveListInt.part.tsx [hunk 1]
```diff
@@ -428,7 +441,8 @@ export class InteractiveListInt extends React.PureComponent<InteractiveListProps
 
   private createItem(key: string, index: number, content: React.ReactChild) {
     const { value, selected } = this.state;
-    const { disabled, showTick = false, multiple = false, condensed = false } = this.props;
+    const { disabled, showTick = false, multiple = false, condensed = false, disabledItems } = this.props;
```
Comment (guiherzog) [inline src/components/InteractiveList/InteractiveListInt.part.tsx]: You can initialize disabled items by default with an array and avoid checking, like it's being done in all other props.

`disabledItems = []`

File: src/components/InteractiveList/InteractiveListInt.part.tsx [hunk 2]
```diff
@@ -60,14 +61,15 @@ const ListWrapper = styled('ul')<InteractiveListWrapperProps>(
...
-    ({ hovered, theme: { ui3, text2 } }) => css`
+    ({ hovered, theme: { ui3, text2 }, disabled }) => css`
       ${getFontStyle({ size: 'medium' })}
       
       background: ${hovered ? ui3 : transparent};
       color: ${text2};
       list-style: none;
       box-sizing: border-box;
-      cursor: pointer;
+      opacity: ${disabled ? '0.5' : '1.0'};
```
Comment (guiherzog) [inline src/components/InteractiveList/InteractiveListInt.part.tsx]: `opacity` is a very expensive attribute to render in css. I would use a background-color or font color instead.

File: src/components/InteractiveList/InteractiveListInt.part.tsx (hunk 1 above)
Comment (guiherzog) [inline src/components/InteractiveList/InteractiveListInt.part.tsx]: You can initialize disabled items by default with an array and avoid checking, like it's being done in all other props.

disabledItems = []

File: src/components/InteractiveList/InteractiveListInt.part.tsx (hunk 2 above)
Comment (lestard) [reply src/components/InteractiveList/InteractiveListInt.part.tsx]: I've used opacity here to match the styling of [Checkbox](https://github.com/ZEISS/precise-ui/blob/develop/src/components/Checkbox/index.tsx#L41) which also uses opacity.

**Pull request merged**